### PR TITLE
Orphanet Alignment for September release - part2

### PR DESCRIPTION
**Pull request by sabrinatoro**
This PR is adding new terms from orphanet. spreadsheet is: [ROBOT partial 3 -new](https://docs.google.com/spreadsheets/d/1QWt2i8Klv6l51vT2ecenLKvdLCF1_ZXWc-q5Jz7JXt4/edit?gid=1321745123#gid=1321745123)

In addition:
- review classification for 'Phelan-McDermid syndrome' (MONDO:0011652)
- children for 'megaloblastic anemia-immunodeficiency due to folate transporter 1 deficiency'

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -559829,6 +559832,40 @@ subset: rare
...
+id: MONDO:0971056
+name: ocular surface squamous neoplasia
+xref: Orphanet:659744 {source="MONDO:equivalentTo"}
+is_a: MONDO:0002532 {source="Orphanet:659744", source="PMID:34424655", source="https://orcid.org/0000-0002-4142-7153"} ! squamous cell neoplasm
+is_a: MONDO:0021220 {source="Orphanet:659744", source="PMID:34424655", source="https://orcid.org/0000-0002-4142-7153"} ! eye neoplasm
+
+[Term]
+id: MONDO:0971058
+name: verruga peruana
+def: "A rare bacterial infectious disease characterized by development of endothelial cell-derived, blood-filled tumors on the surface of the skin during the chronic phase of infection with <i>Bartonella bacilliformis</i>. The lesions can persist for months to years and may cause pain and scarring due to ulceration. They are classified as miliary (multiple red papules of <3 mm in diameter), mular (blood-filled nodules), and diffuse (groups of subdermal nodules >5 mm in diameter). Eruptions may be accompanied by fever and malaise, lymphadenopathy, acute bone and joint pains, and headache. Fatality rates during this phase are low." [Orphanet:659759]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:559864]: should remove the html tags (<i>,</i>)

**Review by nicolevasilevsky — APPROVED** [review]
Two minor changes needed. I approved this though so you can merge without waiting for my re-review

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -263095,16 +263096,16 @@ subset: ordo_malformation_syndrome {source="Orphanet:48652"}
...
 synonym: "22q13.3 deletion syndrome" RELATED [GARD:0010130]
 synonym: "chromosome 22Q13.3 deletion syndrome" RELATED [OMIM:606232]
 synonym: "deletion 22q13.3 syndrome" RELATED [GARD:0010130]
-synonym: "monosomy 22q13" EXACT []
-synonym: "monosomy type 22q13" EXACT [MONDORULE:7, Orphanet:48652]
+synonym: "monosomy 22q13" NARROW []
+synonym: "monosomy type 22q13" NARROW [MONDORULE:7, Orphanet:48652]
 synonym: "Phelan McDermid syndrome" EXACT [PMID:28346892]
-synonym: "PHELAN-McDermid syndrome" RELATED [MONDO:Lexical, OMIM:606232]
+synonym: "PHELAN-McDermid syndrome" EXACT [MONDO:Lexical, OMIM:606232]
```
[the remark this answers is not in the document]
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:263106]: addressed

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:559864]: addressed

**Review by nicolevasilevsky — APPROVED** [review]

**Pull request merged**